recipes/email2pdf/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "email2pdf" %}
+{% set version = "0.9.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2f7d1cbd2e2e41ccd1ab46b6ada586fe4aed6be77507b225d4966b02511b15bc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - beautifulsoup4
+    - python-magic
+    - chardet
+    - pypdf2
+    - html5lib
+    - reportlab
+    - lxml
```

Comment:
Upstream lists different dependencies, are they correct?

https://github.com/andrewferrier/email2pdf/blob/c3b20226bc255a75f52c762aece66c58fb76b2c4/setup.py#L12-L17

Reply:
Fixed the dependencies